src/test/regress/pg_regress_multi.pl

```diff
@@ -486,6 +486,13 @@ sub generate_hba
 # we disable slow start by default to encourage parallelism within tests
 push(@pgOptions, "citus.executor_slow_start_interval=0ms");
 
+# we set some GUCs to not break postgres vanilla tests
+if($vanillatest)
```

Comment:
can we move this to PreVanillaTest?

Reply:
server is already be started if I move it there, so it will not set the option for the server.